app/controllers/games_controller.rb

```diff
@@ -26,6 +29,14 @@ def summary
     end
   end
 
+  def resumes
+    send_data create_zip_of('resume').read, filename: 'resumes.zip'
+  end
+
+  def transcripts
+    send_data create_zip_of('transcript').read, filename: 'transcripts.zip'
+  end
+
```

Comment:
How long can we expect these two actions to take? 
Do we want to background these zip tasks so we don't steal a web worker for a long amount of time?

Reply:
Not actually sure, it shouldn't be horribly slow. I figured if it is too slow during real-world usage then we could just refactor this accordingly. Nothing is written to the filesystem and this is all done in ram so it shouldn't be horribly slow.